tests/components/homematicip_cloud/test_climate.py

```diff
@@ -633,6 +633,33 @@ async def test_hmip_climate_services(
     # There is no further call on connection.
     assert len(home._connection.mock_calls) == 10
 
+    await hass.services.async_call(
+        "homematicip_cloud",
+        "set_cooling",
+        {"accesspoint_id": HAPID, "cooling": False},
+        blocking=True,
+    )
+    assert home.mock_calls[-1][0] == "set_cooling"
+    assert home.mock_calls[-1][1] == (False,)
+    assert len(home._connection.mock_calls) == 11
+
+    await hass.services.async_call("homematicip_cloud", "set_cooling", blocking=True)
```

Comment:
I don't understand, why the "true" value for cooling is missing?

Reply:
The cooling value is optional. Default is True

Reply:
Fixed

Reply:
Added an extra test for true/false/no value